.github/workflows/lint.yml

```diff
@@ -3,6 +3,9 @@ name: lint
 
 on: [pull_request]
 
+permissions:  # added using https://github.com/step-security/secure-workflows
```

Comment:
@jbampton could you describe the added value for adding this permissions to this workflow?

Reply:
@teresaromero the default permissions are read/write, this limits the scope of our actions to what we really need. It also makes it more obvious. I think we should go with this.